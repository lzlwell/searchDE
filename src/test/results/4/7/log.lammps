LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.2674 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1076.8865   -1076.8865      17.1317           10           50 
       1   -1121.3772   -1121.3772      17.1317           10           50 
       2   -1142.1193   -1142.1193      17.1317           10           50 
       3   -1154.6188   -1154.6188      17.1317           10           50 
       4   -1166.6765   -1166.6765      17.1317           10           50 
       5   -1173.0189   -1173.0189      17.1317           10           50 
       6   -1179.4217   -1179.4217      17.1317           10           50 
       7   -1182.9769   -1182.9769      17.1317           10           50 
       8   -1185.8015   -1185.8015      17.1317           10           50 
       9   -1187.8536   -1187.8536      17.1317           10           50 
      10   -1189.8972   -1189.8972      17.1317           10           50 
Loop time of 0.36006 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1076.88649336     -1187.85362181       -1189.897195
  Force two-norm initial, final = 428.815 21.6643
  Force max component initial, final = 296.49 15.2114
  Final line search alpha, max atom move = 0.0118869 0.180816
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.142255 (39.5088)
Neigh time (%) = 0.214357 (59.5338)
Comm  time (%) = 0.00143266 (0.397894)
Outpt time (%) = 0.000132322 (0.0367501)
Other time (%) = 0.00188231 (0.522778)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2361 ave 2361 max 2361 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46944 ave 46944 max 46944 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46944
Ave neighs/atom = 156.48
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
