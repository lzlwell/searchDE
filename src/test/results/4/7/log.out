LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.2674 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1076.8865   -1076.8865      17.1317           10           50 
       1   -1121.3772   -1121.3772      17.1317           10           50 
       2   -1142.1193   -1142.1193      17.1317           10           50 
       3   -1154.6188   -1154.6188      17.1317           10           50 
       4   -1166.6765   -1166.6765      17.1317           10           50 
       5   -1173.0189   -1173.0189      17.1317           10           50 
       6   -1179.4217   -1179.4217      17.1317           10           50 
       7   -1182.9769   -1182.9769      17.1317           10           50 
       8   -1185.8015   -1185.8015      17.1317           10           50 
       9   -1187.8536   -1187.8536      17.1317           10           50 
      10   -1189.8972   -1189.8972      17.1317           10           50 
Loop time of 0.36006 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1076.88649336     -1187.85362181       -1189.897195
  Force two-norm initial, final = 428.815 21.6643
  Force max component initial, final = 296.49 15.2114
  Final line search alpha, max atom move = 0.0118869 0.180816
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.142255 (39.5088)
Neigh time (%) = 0.214357 (59.5338)
Comm  time (%) = 0.00143266 (0.397894)
Outpt time (%) = 0.000132322 (0.0367501)
Other time (%) = 0.00188231 (0.522778)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2361 ave 2361 max 2361 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46944 ave 46944 max 46944 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46944
Ave neighs/atom = 156.48
Neighbor list builds = 10
Dangerous builds = 10
